tests/test_market.py

```diff
@@ -637,31 +630,13 @@ def check_avg_trade_price(self):
     @precondition(lambda self: self.market.traded_energy)
     @rule()
     def check_acct(self):
-        actor_sums = defaultdict(int)
```

Comment:
Good that you replaced it here too, hope it did not give you a hard time to fix the 'hypothesis' tests.

Reply:
No, no problems actually.